cmd/juju/system/createenvironment.go

```diff
@@ -22,15 +22,19 @@ import (
 	localProvider "github.com/juju/juju/provider/local"
 )
 
-// CreateEnvironmentCommand calls the API to create a new environment.
-type CreateEnvironmentCommand struct {
+func newCreateEnvironmentCommand() cmd.Command {
+	return envcmd.WrapSystem(&createEnvironmentCommand{})
+}
+
+// createEnvironmentCommand calls the API to create a new environment.
+type createEnvironmentCommand struct {
 	envcmd.SysCommandBase
 	api CreateEnvironmentAPI
 
-	name       string
-	owner      string
-	configFile cmd.FileVar
-	confValues map[string]string
+	Name       string
```

Comment:
Let's be consistent: either use the export_test approach we've used elsewhere, or use the exported field thing as here (with a comment saying that the fields are exported for tests)